src/mock_crust/support.rs

```diff
@@ -294,6 +287,12 @@ impl<UID: Uid> ServiceHandle<UID> {
             .is_peer_connected(&unwrap!(handle.0.borrow().uid))
     }
 
+    /// Delivers all messages that are queued in the network.
+    pub fn poll_network(&self) {
```

Comment:
would it maybe help to rename this and the `Network::poll` to something like dispatch / deliver messages instead. Just so we can keep the poll functions exclusive to the handling events part of the mock crust flow.